bindgen/templates/RecordTemplate.cs

```diff
@@ -6,6 +6,17 @@
 {%- let (ordered_fields, is_reordered) = rec.fields()|order_fields %}
 
 {%- call cs::docstring(rec, 0) %}
+{%- for field in ordered_fields %}
+{%- match field.docstring() %}
+{%- when Some with(docstring) %}
+/// <param name="{{ field.name() }}">
+{%- for line in docstring.lines() %}
+/// {{ textwrap::dedent(line) }}
```

Comment:
Looking at this code again, `dedent` should happen on the whole docstring, not individual lines. Dedenting individual lines would interrupt indentation within the docstring.
```suggestion
{%- let docstring = textwrap::dedent(docstring) %}
{%- for line in docstring.lines() %}
/// {{ line }}
```